SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/entity/Customer.java

```diff
@@ -62,7 +62,7 @@ public void setCustomerID(int customerID) {
         this.ID = customerID;
     }
  
-    @Column(name = "Password", length = 20, nullable = false)
+    @Column(name = "password", length = 45, nullable = false)
```

Comment:
I think length should be 200.

Reply:
I think 200 characters is a bit excessive for the scope of our app.

Reply:
'twas joking...